### ISSUE TITLE
Provide quickstarter metadata in release-manager.yml

### ISSUE THREAD

**Issue by fadivr** [opendevstack/ods-quickstarters#75]
To provide ods component properties in code
we would need the following addtional informations in the release-manager.yml file:

- Identification
- Name of Software, 
- Supplier, 
- Version, 
- Description of Functionality, 
- References

Those shall be used in the generation of "System Design Specification" Document (DSD) section 5.2

**Comment by metmajer** [comment]
@fadivr please provide some examples on `Identification` and `References`

**Comment by metmajer** [comment]
I have clarified and reduced the number of properties needed so they serve as a mandatory, yet a generic description of an ODS component. Those properties are: `name`, `description`, `supplier`, `version`. Here is an example of how I use this in a demo application:

```
---
metadata:
  name: "Sock Shop: Shopping Cart"
  description: The shopping cart service of a socks-selling e-commerce demo application.
  supplier: https://github.com/microservices-demo/carts
  version: 0.4.8
```

@michaelsauter to not break the pipelines (as these properties must exist), we should provide these fields in every quickstarter. How do you want me to proceed? Options:

a) Have me create a PR that provides a suggestion for each quickstarter to be reviewed
b) Bring this to the quickstarter owners directly

Thanks!

**Comment by michaelsauter** [comment]
@metmajer Cool. Right now we do not have a defined owner for each quickstarter. We can either:

* distribute work among a few people that maintain most quickstarters
* first assign one owner for each quickstarter for ODS 3, which I am hoping to have in January

When should this change land?

**Comment by metmajer** [comment]
@michaelsauter I have just seen that it's not too many quickstarters anyway. Therefore, I would provide a PR and have the people you recommend comment on it. Probably more efficient that way.

Regarding the timing, I'll contact you elsewhere.

**Comment by metmajer** [comment]
Closed via #144
